demo.html

```diff
@@ -61,6 +67,51 @@
     <h1 slot="site-title">Calendar Component Demo</h1>
   </byu-header>
   <div class="page-content">
+    
+    <h1 class="subheader">List View</h1>
+    <byu-calendar 
+      title="Upcoming Events"
+      categories="3+4+5"
+      days="7"
+      price="4.5"
+      display="1"
+      limit="4">
+    </byu-calendar>
+    <hr>
+
+    <h1 class="subheader">Vertical Tiles</h1>
+    <byu-calendar 
+      title="Upcoming Events"
+      categories="3+4+5"
+      days="7"
+      price="4.5"
+      display="2"
+      limit="4">
+    </byu-calendar>
+    <hr>
+
+    <h1 class="subheader">Horizontal Tiles</h1>
+    <byu-calendar 
+      title="Upcoming Events"
+      categories="3+4+5"
+      days="7"
+      price="4.5"
+      display="3"
+      limit="4">
+    </byu-calendar>
+    <hr>
+    
+    <h1 class="subheader">Horizontal Tiles (No limit)</h1>
+    <byu-calendar 
+      title="Upcoming Events"
+      categories="3+4+5"
+      days="7"
+      price="4.5"
+      display="3">
```

Comment:
I noted above asking about the limit variable being optional. I thought the line had to be formatted differently, but it is looking like it works as it is. I will let someone else decide if the line (defining limit variable originally) should be changed.